grapher/core/Grapher.jsdom.test.ts

```diff
@@ -0,0 +1,16 @@
+#! /usr/bin/env yarn jest
+
+import { Grapher } from "grapher/core/Grapher"
+
+describe(Grapher, () => {
+    it("can serialize Grapher for saving", () => {
+        expect(Object.keys(new Grapher().object).length).toBe(0)
+    })
+
+    it("regression fix: container options are not serialized", () => {
+        const grapher = new Grapher({ xAxis: { min: 1 } })
+        const obj = grapher.object.xAxis as any
+        expect(obj?.max).toBe(undefined)
```

Comment:
These tests are weird. Are you testing if `obj` or `obj.max` is undefined? (Because here you're testing for either one)

Reply:
Agreed. Just cleaned it up. Obj is supposed to be defined, max not. Cleaned that up and added a comment about the next text.